docs/rc.xml.all

```diff
@@ -82,19 +82,31 @@
 
     custom format - (introduced in 0.7.2)
     It allows one to replaces all the above "fields" with one line, using a
-    printf style format. The example below would print "foobar",then type of
-    window (wayland, X), then state of window (M/m/F), then output (shows
-    if more than 1 active), then workspace name, then identifier/app-id,
-    then the window title. Uses 100% of OSD window width.
-    Note: printf style formatting uses a subset, "man 3 printf",
-    can insert text, left/right justify, and set field length.
-    Future enhancements may allow more than one custom format.
+    printf style format.
+
+    The example below would print "foobar",then type of window (wayland, X),
+    then state of window (M/m/F), then output (shows if more than 1 active),
+    then workspace name, then identifier/app-id, then the window title.
+    Uses 100% of OSD window width.
 
     <windowSwitcher show="yes" preview="no" outlines="no" allWorkspaces="yes">
       <fields>
         <field content="custom" format="foobar %b %3s %-10o %-20W %-10i %t" width="100%" />
       </fields>
     </windowSwitcher>
+
+
+    Note: printf style formatting using a subset of options, see "man 3 printf"
+
+    Options:
+     random text may be inserted
+     field length, example "%10" use 10 spaces, even if text uses less
+     left jusify text, example "%-"
+     right justify text, example "%" instead of "%-"
+
+    example, %-10 would left justify and make room for 10 charaters
+
+    Future enhancements may allow more than one custom format.
```

Comment:
Suggest replacing this with a reference to the labwc-config manpage.

Reply:
Ok, remove from rc.xml.all and clarified in labwc-config